pure3.md

```diff
@@ -0,0 +1,282 @@
+  an update released by SolarWinds in the spring of 2020, yet it was not
+  reported until December of that year.
+
+Automotive software supply chains are particularly tempting targets for
+attackers. This is because automotive software runs on a large number of
+vehicles and updating all of them to recover quickly from an attack is a
+slow and difficult proposition. As such, a successful software supply chain
+attack means that thousands (or more) of vehicles are impacted. Any attack that
+targets a large number of vehicles on roads can do immeasurable damage. Further,
+while Uptane can securely deliver updates to vehicles to recover from the
+attack, ensuring every vehicle on the road receives the update immediately is
+very difficult given the circumstances of use of these vehicles. Therefore,
+strategies that can secure automotive software supply chains are vital to
+_avoid_ such attacks altogether.
+
+Governments around the world have recognized this fact. Regulations are
+forthcoming from the United Nations and other governing bodies, and standards
+have been published that specify how software running on automobiles must be
+secured. President Biden of the United States of America issued an Executive
+Order describing the need to improve the nation's cybersecurity. This EO, in
+particular, pushes for solutions like Software Bills of Materials (SBOMs) that
+list the components of software artifacts. This information is vital in ensuring
+developers are aware of the components in their supply chain so that they can
+fix vulnerabilities as and when they are discovered.
+
+Unfortunately, SBOMs do not provide end-to-end software supply chain security.
+For example, an SBOM would not have prevented an attack that used a compromised
+build server as seen in the SolarWinds hack. A more comprehensive framework is
+necessary and is described in this document.
+
+## Reasoning
+
+in-toto is a comprehensive framework that secures each step in the software
+supply chain. With in-toto, software supply chain owners can enforce policies
+pertaining to the flow of artifacts through the supply chain, and provide proof
+that authorized people or bots made changes to artifacts at any given step.
+Thus, in-toto can be used to secure software from the very first step all the
+way through the development pipeline. Furthermore, the framework plays well with
+other systems and tools that attempt to secure specific aspects of the software
+supply chain. in-toto is integrated into build tools and systems like Jenkins
+and Tekton, and is compatible with complementary software supply chain security
+efforts such as Grafeas, Sigstore, and Software Bills of Materials.
+
+## Specification
+
+Note: The reader is expected to be familiar with the
+[in-toto Specification](https://github.com/in-toto/docs/blob/master/in-toto-spec.md).
+
+We propose in-toto as an option to achieve software supply chain security
```

Comment:
as an option?  Do you mean something more like: We describe how in-toto could be used to achieve...